src/test/java/pl/coderslab/service/DriverServiceTest.java

```diff
@@ -0,0 +1,111 @@
+package pl.coderslab.service;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.runners.MockitoJUnitRunner;
+import pl.coderslab.model.Cargo;
+import pl.coderslab.model.Driver;
+import pl.coderslab.repository.DriverRepository;
+
+import javax.validation.constraints.AssertTrue;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DriverServiceTest {
+
+    @Mock
+    private DriverRepository driverRepository;
+
+    @InjectMocks
+    private DriverService driverService;
+
+    @Test
+    public void shouldAdd() {
```

Comment:
Właściwie jeżeli korzystasz z mocków to wystarczyłoby zamienić DriverRepository na Spy zamiast mock i sprawdzić czy został wywołany z twoim argumentem. A tego typu sprawdzanie przeniesć w przyszłości do testów integracyjnych z bazą danych w pamięci (H2).